src/exec.h

```diff
@@ -4,7 +4,33 @@
 #include <boost/asio/io_context.hpp>
 #include <boost/filesystem.hpp>
 #include <boost/format.hpp>
+
+#include <boost/version.hpp>
+
+#if BOOST_VERSION >= 108800
```

Comment:
I wonder why they didn't do such "#if/#else" in the boost itself for the `boost::process` namespace.